pkg/kubelet/rkt/rkt.go

```diff
@@ -51,8 +51,8 @@ import (
 )
 
 const (
-	acVersion             = "0.6.1"
-	rktMinimumVersion     = "0.8.1"
+	acVersion             = "0.7.1"
+	supportedRktVersion   = "0.9.0"
```

Comment:
@yujuhong We should guarantee 0.9.1 won't break 0.9.0, but not 0.10.0 afaics. If 0.10.0 will fail in an unexpected way, I think we should just abort here?
Maybe we can loose the restriction a little bit, say support all 0.9.x? 
@jonboulle